cmd/build/v2/build.go

```diff
@@ -131,28 +144,49 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 
 	buildManifest := options.Manifest.Build
 
-	for _, svcToBuild := range toBuildSvcs {
-		if options.EnableStages {
-			oktetoLog.SetStage(fmt.Sprintf("Building service %s", svcToBuild))
-		}
+	oktetoLog.Infof("Images to build: [%s]", strings.Join(toBuildSvcs, ", "))
+	for len(bc.builtImages) != len(toBuildSvcs) {
```

Comment:
If an image build returns an error we'll skip this loop. We can't iterate through `toBuildSvc` because we can't ensure that the order provided by the map is the correct one and ordering them requires creating an algorithm that can be more resources demanding than iterating over the loop until we finish all the image build